### PR TITLE
Fix crates

### PR DESCRIPTION
**Pull request by piobab**

### CODE REVIEW

**Review by dancreee — APPROVED** [review]

**Review by grod220 — APPROVED** [review]

File: contracts/red-bank/Cargo.toml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [package]
 name          = "mars-red-bank"
+description   = "The Red Bank exposes all interactions that are done between the user and the money market"
```
Comment (larry0x) [inline contracts/red-bank/Cargo.toml:3]: This sentence doesn't make sense to be

File: contracts/rewards-collector/osmosis/Cargo.toml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 [package]
 name          = "mars-rewards-collector-osmosis"
+description   = "Manages protocol revenue in various coins from the Red Bank"
```
Comment (larry0x) [inline contracts/rewards-collector/osmosis/Cargo.toml:3]:
```suggestion
description   = "A smart contract that manages protocol revenue in various coins from the Red Bank"
```

File: contracts/red-bank/Cargo.toml (hunk 1 above)
Comment (larry0x) [reply contracts/red-bank/Cargo.toml:3]:
```suggestion
description   = "A smart contract that manages asset deposit, borrowing, and liquidations"
```

**Comment by larry0x** [comment]
@piobab @dancreee It seems that the clicking "commit suggestion" button makes the commit to not be signed, which prevents the PR from being merged.

**Comment by piobab** [comment]
> @piobab @dancreee It seems that the clicking "commit suggestion" button makes the commit to not be signed, which prevents the PR from being merged.

No, it's my fault. I need to debug but my signing disable from time to time. I pushed fix.

**Pull request merged**